p/vsn/index.js

```diff
@@ -11,6 +11,21 @@ const transformReqBody = (ctx, body) => {
 	return body
 }
 
+const formatRefreshJourneyReq = (ctx, refreshToken) => {
+	const {profile, opt} = ctx
+
+	return {
+		meth: 'Reconstruction',
+		req: {
+			outReconL: [{ctx: refreshToken}],
```

Comment:
It would be very interesting to know wether this is a change that all HAFAS endpoints with e.g. version `1.24`, or wether this is VSN-specific.

Do you know @marudor?